_providers/mailbox.org.md

```diff
@@ -1,6 +1,7 @@
 ---
 name: Mailbox.org
-status: OK
+
```

Comment:
This empty line is redundant, better delete it. Not sure if it breaks anything, but better be sure.

The format of this file is YAML, and it can be quite tricky in what is "normal text" and what isn't.

Reply:
Thanks for the feedback! I deleted it but saw that it is there in other providers (also gmx), weird...